index.html

```diff
@@ -30,68 +30,72 @@
+                </hgroup>
             </div>
-        </form>
-        <div id="search-results">
-            <h3>Search Results</h3>
-            <ul></ul>
-        </div>
-        <div id="zen-error" class="hide flash flash-error"></div>
-        <!-- The video, along with related content & messages -->
-        <div id="audioplayer">
-            <h3>
-                <a id="zen-video-title" class="plyr__cite plyr__cite--video"></a>
-            </h3>
-
-            <div id="playerControls">
-                <button class="btn" id="togglePlayer">
-                    Show Player
-                </button>
-                <button class="btn" id="toggleDescription">
-                    Show Description
-                </button>
-                <input id="toggleRepeat" name="toggleRepeat" type="checkbox" />
-                <label id="toggleRepeatLabel" for="toggleRepeat">Repeat track</label>
+        </header>
+        <div id="container">
+            <!-- The form -->
+            <form id="form">
+                <div class="input-group">
+                    <input type="text" id="v" name="v" class="input-search color-grey" placeholder="Search, YouTube video ID, or URL">
+                    <span class="input-group-button">
+                        <button class="btn btn-search" id="submit">
+                            <i class="fa fa-search"></i>
+                        </button>
+                    </span>
+                </div>
+            </form>
+            <div id="search-results">
+                <h3>Search Results</h3>
+                <ul></ul>
             </div>
+            <div id="zen-error" class="hide flash flash-error"></div>
+            <!-- The video, along with related content & messages -->
+            <div id="audioplayer">
+                <h3>
+                    <a id="zen-video-title" class="plyr__cite plyr__cite--video"></a>
+                </h3>
 
-            <!-- Plyr controls' icons and Plyr player itself -->
-            <div id="plyr-svg" hidden>
-            </div>
-            <div class="plyr">
-                <audio></audio>
+                <div id="playerControls">
+                    <button class="btn" id="togglePlayer">
+                        Show Player
+                    </button>
+                    <button class="btn" id="toggleDescription">
+                        Show Description
+                    </button>
+                    <input id="toggleRepeat" name="toggleRepeat" type="checkbox" />
+                    <label id="toggleRepeatLabel" for="toggleRepeat">Repeat track</label>
+                </div>
+
+                <!-- Plyr controls' icons and Plyr player itself -->
+                <div id="plyr-svg" hidden>
+                </div>
+                <div class="plyr">
+                    <audio></audio>
+                </div>
+                <br>
+                <div id="zen-video-description"></div>
```

Comment:
The content in this div is now unreadable with black on dark blue text

<img width="519" alt="description" src="https://user-images.githubusercontent.com/1480063/31479056-a5551340-aec9-11e7-832b-ce900336f85c.png">